### PR TITLE
handle not connected to kbfs better

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers this fixes issues @malgorithms was seeing w/ incorrect state being presented when you don't have a good connection to kbfs locally.

We set out bootstrapping state to loaded when we got the initial payload from the server, but we're still partway through the process. we still need to wait for kbfs to connect ,then we're really in.

I added comments to this file about us needing to saga-ize it. The majority of the code this thing has is cause we didn't have sagas. Doing that will clean this thing up a ton and reduce all the wackiness.

Until then, some more wackiness thrown on top

appropriate image

### CODE REVIEW

File: shared/actions/config/index.js [hunk 1]
```diff
@@ -119,14 +135,15 @@ const bootstrap = (opts?: BootstrapOptions = {}): AsyncAction => (dispatch, getS
       console.log('[bootstrap] bootstrapping on connect')
       dispatch(bootstrap())
     })
-    engine().listenOnDisconnect('daemonError', () => {
```
Comment (chrisnojima) [inline shared/actions/config/index.js]: have to move this down else we flash the disconnect when we actually disconnect between retries

File: shared/actions/config/index.js [hunk 2]
```diff
@@ -159,7 +176,7 @@ const getBootstrapStatus = (): AsyncAction => dispatch => (
 
         dispatch({
           payload: {bootstrapStatus},
-          type: Constants.bootstrapLoaded,
+          type: Constants.bootstrapStatusLoaded,
```
Comment (chrisnojima) [inline shared/actions/config/index.js]: we didn't bootstrap, we got part of it

File: shared/login/forms/index.desktop.js [hunk 3]
```diff
@@ -5,11 +5,11 @@ import {globalColors, globalStyles, globalMargins} from '../../styles'
 
 import type {Props} from '.'
 
-const Splash = () => (
+const Splash = (props: Props) => (
   <Box style={stylesLoginForm}>
     <Icon type='icon-keybase-logo-128' />
     <Text style={stylesHeader} type='HeaderBig'>Keybase</Text>
-    <Text style={stylesHeaderSub} type='BodySmall'>Loading…</Text>
+    <Text style={stylesHeaderSub} type='BodySmall'>Loading…{props.retrying ? ' (still trying)' : ''}</Text>
```
Comment (chrisnojima) [inline shared/login/forms/index.desktop.js]: give a little hint we're still doing stuff

**Comment by chrisnojima** [comment]
@chromakode fyi

File: shared/actions/config/index.js (hunk 2 above)
Comment (chromakode) [inline shared/actions/config/index.js]: Good catch.

**Pull request merged**